scipy/signal/windows/_windows.py

```diff
@@ -1716,8 +1717,8 @@ def taylor(M, nbar=4, sll=30, norm=True, sym=True):
     Parameters
     ----------
     M : int
-        Number of points in the output window. If zero or less, an
-        empty array is returned.
+        Number of points in the output window. If zero an empty array is returned.
+        An exception thrown when it is negative.s
```

Comment:
The last `s` is typo?
```suggestion
        An exception is thrown when it is negative.
```